error.go

```diff
@@ -3,26 +3,37 @@ package apns
 import (
 	"bytes"
 	"encoding/binary"
-	"errors"
 )
 
 var (
-	ErrDisconnected = errors.New("disconnected from gateway")
+	// ErrUnrecognizedErrorResponse when the error from Apple isn't recognized.
+	ErrUnrecognizedErrorResponse = "Unrecognized error or no error."
 )
 
 const (
 	// Error strings based on the codes specified here:
 	// https://developer.apple.com/library/ios/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/Chapters/CommunicatingWIthAPS.html#//apple_ref/doc/uid/TP40008194-CH101-SW12
-	ErrProcessing         = "Processing error"
+
+	// ErrProcessing (1)
+	ErrProcessing = "Processing error"
```

Comment:
If these are going to be exported and called Err*, they should probably be errors instead of strings.

But then these are just part of the Error struct, no the entire error.